src/dsp_tools/models/project.py

```diff
@@ -472,44 +471,9 @@ def delete(self) -> Project:
         :return: DSP response
         """
 
-        result = self._con.delete(Project.IRI + quote_plus(self._id))
+        result = self._con.delete(Project.IRI + quote_plus(self._iri))
         return Project.fromJsonObj(self._con, result["project"])
 
-    def set_default_permissions(self, group_id: str) -> None:
-        permobj = {
-            "forGroup": "http://www.knora.org/ontology/knora-admin#ProjectMember",
-            "forProject": self._id,
-            "hasPermissions": [
-                {
-                    "additionalInformation": None,
-                    "name": "ProjectResourceCreateAllPermission",
-                    "permissionCode": None,
-                }
-            ],
-        }
-        jsondata = json.dumps(permobj, indent=4)
-        print(jsondata)
-        result = self._con.post("/admin/permissions/ap", jsondata)
-        pprint(result)
-
-        return
-        permobj = {
```

Comment:
unreachable code after `return`. Then I realized the entire function is never used. 

(Might be interesting because it handles permissions. But if we implement permissions handling in dsp-tools, I prefer starting over from zero, rather than working with such pirate's code...)

Reply:
understandable ^^

but I think this gives a good hint for one option of where permissions could be added. So maybe it's worth keeping a permalink to this version of the file, so that you can look at it again, when/if you implement permissions in dsp-tools

Reply:
Good idea 👍